server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/UploadStepdefs.java

```diff
@@ -109,14 +111,24 @@ public void userUploadContent(String username) throws Throwable {
             .bodyStream(new BufferedInputStream(new ZeroedInputStream(_1M), _1M), org.apache.http.entity.ContentType.DEFAULT_BINARY);
 
         request.addHeader("Authorization", accessToken.serialize());
-        response = Executor.newInstance(HttpClientBuilder.create().disableAutomaticRetries().build()).execute(request).returnResponse();
+        response = Executor.newInstance(newClient())
+            .execute(request)
+            .returnResponse();
     }
 
     @When("^someone upload a content without authentification$")
     public void userUploadContentWithoutAuthentification() throws Throwable {
         Request request = Request.Post(uploadUri)
-            .bodyStream(new BufferedInputStream(new ZeroedInputStream(_1M), _1M), org.apache.http.entity.ContentType.DEFAULT_BINARY);
-        response = Executor.newInstance(HttpClientBuilder.create().disableAutomaticRetries().build()).execute(request).returnResponse();
+            .bodyStream(new BufferedInputStream(new ZeroedInputStream(_24K), _24K), org.apache.http.entity.ContentType.DEFAULT_BINARY);
```

Comment:
why did you reduced it so much?

Reply:
Did you read the commit message?

Honestly I believe that here we were saying:
 - I'm testing a "feature": non authenticated people get kicked

While what we were testing was:
 - I'm testing a "feature": non authenticated people uploading laaaarge content get kicked

(this is not what the gerkins spec described...)

I'm fixing the case that was described, thus **fixing master**.

We can plan to add work (asynchronously) to actually test `non authenticated people uploading laaaarge content get kicked` but I believe that 1. it should be another test 2. it do not deserve keeping master broken.